scripts/db_migrations/v2.18/upload_models_s3.py

```diff
@@ -0,0 +1,66 @@
+import json
+import gridfs
+import sys
+import uuid
+from pymongo import MongoClient
+import boto3
+
+##### Connect to AWS S3 #####
+s3 = boto3.client('s3')
+bucket_name = '3drepo-models'
+
+mongoURL = sys.argv[1]
+mongoPort = sys.argv[2]
+userName = sys.argv[3]
+password = sys.argv[4]
+connString = "mongodb://"+ userName + ":" + password +"@"+mongoURL + ":" + mongoPort + "/"
+##### Enable dry run to not commit to the database #####
+dry_run = True
+
+##### Connect to the Database #####
+def upload(col_prefix, col_insert_prefix, file_type):
+	db = MongoClient(connString)
+	for database in db.database_names():
+		if database != "admin" and database != "local" and database == "kierongibbons":
+			db = MongoClient(connString)[database]
+			print(file_type + ": --database:" + database)
+
+##### Get a model ID #####
+			for settings in db.settings.find():
+				model_id = settings.get('_id')
+				col_name = model_id + col_prefix
+				fs = gridfs.GridFS(db, col_name)
+				fs_model = fs.find()
+				for model in fs_model:
```

Comment:
merge with line above, i.e. `for model in fs.find()`

don't call this `model`, you're looping through files. call it `file` or `entry` or whatever. Model is too widely use in our systems